src/__tests__/Toast.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import * as _solidIcons from '@fortawesome/free-solid-svg-icons';
+import { mount } from 'enzyme';
+import 'jest-enzyme';
+import * as React from 'react';
+
+import { IconLibrary } from '../Icon';
+import { NotificationType, Toast } from '../Toast';
+
+function expectToast(type: NotificationType, iconClassName: string) {
+  const toast = mount(
+    <Toast
+      notification={{
+        title: 'Title',
+        body: 'Body',
+        type,
+      }}
+    />
+  );
+
+  expect(toast.find(`svg.${iconClassName}`)).toHaveLength(1);
+  expect(toast.find('b').text()).toEqual('Title');
+  expect(toast.find('p').text()).toEqual('Body');
```

Comment:
```suggestion
  expect(toast.find('p').text()).toEqual('Body');
  wrapper.unmount();
```

make sure to cleanup.

Also, you can use `toHaveText` matcher.

Reply:
Still left to do ^